ghcide/exe/Main.hs

```diff
@@ -142,4 +147,10 @@ main = withTelemetryLogger $ \telemetryLogger -> do
                 , optCheckProject = pure $ checkProject config
                 , optRunSubset = not argsConservativeChangeTracking
                 }
+#ifdef MONITORING_EKG
+        , IDEMain.argsMonitoring = OpenTelemetry.monitoring <> EKG.monitoring logger argsMonitoringPort
```

Comment:
This is inconsistent with `ghcide/src/Development/IDE/Main.hs`. I think they should probably be the same, and that one looks nicer?

Reply:
In what way is it inconsistent? The one in `ghcide/src/Development/IDE/Main.hs` is setting the default value, and this one is overriding it to customise the port number

Reply:
Oh I see. I thought they were both setting it.